### PR TITLE
[Feature] [OpenAPI] Support OpenAPI 3 securitySchemes

### PR DESCRIPTION
**Pull request by eliasmeire**
## Add support for importing securitySchemes from OpenApi v3

Specification: https://swagger.io/docs/specification/authentication/

This PR partially implements #1698

### This PR adds support for the following Open API security types
- HTTP Basic authentication ([docs](https://swagger.io/docs/specification/authentication/basic-authentication/))
- API keys ([docs](https://swagger.io/docs/specification/authentication/api-keys/))
- Cookies ([docs](https://swagger.io/docs/specification/authentication/cookie-authentication/))

### These types are not handled (yet)
- Bearer ([docs](https://swagger.io/docs/specification/authentication/bearer-authentication/))
- OAuth 2.0 ([docs](https://swagger.io/docs/specification/authentication/oauth2/))
- OpenID ([docs](https://swagger.io/docs/specification/authentication/openid-connect-discovery/))

### CODE REVIEW

File: packages/insomnia-importers/src/__tests__/fixtures.test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('Fixtures', () => {
       const prefix = input.replace(/-input\.[^.]+/, '');
       const output = `${prefix}-output.json`;
 
-      if (!prefix.includes('simple')) {
```
Comment (eliasmeire) [inline packages/insomnia-importers/src/__tests__/fixtures.test.js]: Not sure why this is here, but it was skipping all the open api fixtures. I changed the logic to only skip that failing curl test. Let me know if this needs changes :smile:

Comment (gschier) [reply packages/insomnia-importers/src/__tests__/fixtures.test.js]: Ah, woops. I must have accidentally committed this. What you've done looks good :)

File: packages/insomnia-importers/src/importers/openapi3.js [hunk 2]
```diff
@@ -224,6 +248,100 @@ function prepareHeaders(endpointSchema) {
...
+    });
+
+  if (apiKeyCookies.length > 0) {
+    apiKeyHeaders.push(apiKeyCookieHeader);
+  }
+
+  const httpAuth = supportedSchemes.find(
+    scheme => scheme.type === SECURITY_TYPE.HTTP && scheme.scheme === 'basic',
+  )
+    ? { username: '{{ httpUsername }}', password: '{{ httpPassword }}' }
```
Comment (gschier) [inline packages/insomnia-importers/src/importers/openapi3.js]: The authentication attribute also requires a `type` parameter so this should have `type: 'basic'`. 

Here's a list of accepted authentication types:

```js
AUTH_NONE = 'none';
AUTH_OAUTH_2 = 'oauth2';
AUTH_OAUTH_1 = 'oauth1';
AUTH_BASIC = 'basic';
AUTH_DIGEST = 'digest';
AUTH_BEARER = 'bearer';
AUTH_NTLM = 'ntlm';
AUTH_HAWK = 'hawk';
AUTH_AWS_IAM = 'iam';
AUTH_NETRC = 'netrc';
AUTH_ASAP = 'asap';
```

Comment (eliasmeire) [reply packages/insomnia-importers/src/importers/openapi3.js]: Added `"type":  "basic"` 😄

**Comment by Lyra2108** [comment]
Thanks from me as well :)

**Pull request merged**